File: Data Structures & Algorithms/missing-number/submission-2.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int x, sum = 0, n = nums.size();
        for (int i = 0; i<n;i++){
            sum += nums[i];
        }
        x = (n*(n+1) / 2) - sum;
        return x;
    }
};

File: Data Structures & Algorithms/remove-element/submission-2.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        lastIdx = len(nums)-1
        while lastIdx >= 0 and nums[lastIdx] == val:
            lastIdx -= 1
        
        i = 0
        while i < lastIdx:
            if nums[i] == val:
                nums[i], nums[lastIdx] = nums[lastIdx], nums[i]
                while lastIdx >= 0 and nums[lastIdx] == val:
                    lastIdx -= 1
            i+=1
        return lastIdx + 1